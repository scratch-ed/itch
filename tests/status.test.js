const { runTest, testStatuses } = require('./integration-runner');

jest.setTimeout(20000);

test('01.space_mission', () => {
  return runTest(`status/${expect.getState().currentTestName}`)
    .then(result => {
      expect(result).toMatchSnapshot();
      expect(testStatuses(result)).allStatusesAre('correct');
    });
});