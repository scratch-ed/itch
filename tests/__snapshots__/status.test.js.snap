// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`01.space_mission 1`] = `
Array [
  Object {
    "command": "start-judgement",
  },
  Object {
    "command": "start-tab",
    "title": "Testen uit het testplan",
  },
  Object {
    "command": "start-context",
  },
  Object {
    "command": "start-testcase",
    "description": "Niet geprutst aan vakjes",
  },
  Object {
    "command": "start-test",
    "expected": "false",
  },
  Object {
    "command": "close-test",
    "generated": "false",
    "status": Object {
      "enum": "correct",
      "human": "Juist",
    },
  },
  Object {
    "command": "start-test",
    "expected": "false",
  },
  Object {
    "command": "close-test",
    "generated": "false",
    "status": Object {
      "enum": "correct",
      "human": "Juist",
    },
  },
  Object {
    "command": "start-test",
    "expected": "false",
  },
  Object {
    "command": "close-test",
    "generated": "false",
    "status": Object {
      "enum": "correct",
      "human": "Juist",
    },
  },
  Object {
    "command": "start-test",
    "expected": "false",
  },
  Object {
    "command": "close-test",
    "generated": "false",
    "status": Object {
      "enum": "correct",
      "human": "Juist",
    },
  },
  Object {
    "command": "start-test",
    "expected": "false",
  },
  Object {
    "command": "close-test",
    "generated": "false",
    "status": Object {
      "enum": "correct",
      "human": "Juist",
    },
  },
  Object {
    "command": "start-test",
    "expected": "false",
  },
  Object {
    "command": "close-test",
    "generated": "false",
    "status": Object {
      "enum": "correct",
      "human": "Juist",
    },
  },
  Object {
    "command": "start-test",
    "expected": "false",
  },
  Object {
    "command": "close-test",
    "generated": "false",
    "status": Object {
      "enum": "correct",
      "human": "Juist",
    },
  },
  Object {
    "command": "start-test",
    "expected": "false",
  },
  Object {
    "command": "close-test",
    "generated": "false",
    "status": Object {
      "enum": "correct",
      "human": "Juist",
    },
  },
  Object {
    "command": "start-test",
    "expected": "false",
  },
  Object {
    "command": "close-test",
    "generated": "false",
    "status": Object {
      "enum": "correct",
      "human": "Juist",
    },
  },
  Object {
    "command": "start-test",
    "expected": "false",
  },
  Object {
    "command": "close-test",
    "generated": "false",
    "status": Object {
      "enum": "correct",
      "human": "Juist",
    },
  },
  Object {
    "command": "start-test",
    "expected": "false",
  },
  Object {
    "command": "close-test",
    "generated": "false",
    "status": Object {
      "enum": "correct",
      "human": "Juist",
    },
  },
  Object {
    "command": "start-test",
    "expected": "false",
  },
  Object {
    "command": "close-test",
    "generated": "false",
    "status": Object {
      "enum": "correct",
      "human": "Juist",
    },
  },
  Object {
    "command": "start-test",
    "expected": "false",
  },
  Object {
    "command": "close-test",
    "generated": "false",
    "status": Object {
      "enum": "correct",
      "human": "Juist",
    },
  },
  Object {
    "command": "start-test",
    "expected": "false",
  },
  Object {
    "command": "close-test",
    "generated": "false",
    "status": Object {
      "enum": "correct",
      "human": "Juist",
    },
  },
  Object {
    "command": "start-test",
    "expected": "false",
  },
  Object {
    "command": "close-test",
    "generated": "false",
    "status": Object {
      "enum": "correct",
      "human": "Juist",
    },
  },
  Object {
    "command": "start-test",
    "expected": "false",
  },
  Object {
    "command": "close-test",
    "generated": "false",
    "status": Object {
      "enum": "correct",
      "human": "Juist",
    },
  },
  Object {
    "command": "start-test",
    "expected": "false",
  },
  Object {
    "command": "close-test",
    "generated": "false",
    "status": Object {
      "enum": "correct",
      "human": "Juist",
    },
  },
  Object {
    "command": "start-test",
    "expected": "false",
  },
  Object {
    "command": "close-test",
    "generated": "false",
    "status": Object {
      "enum": "correct",
      "human": "Juist",
    },
  },
  Object {
    "command": "start-test",
    "expected": "false",
  },
  Object {
    "command": "close-test",
    "generated": "false",
    "status": Object {
      "enum": "correct",
      "human": "Juist",
    },
  },
  Object {
    "command": "start-test",
    "expected": "false",
  },
  Object {
    "command": "close-test",
    "generated": "false",
    "status": Object {
      "enum": "correct",
      "human": "Juist",
    },
  },
  Object {
    "command": "start-test",
    "expected": "false",
  },
  Object {
    "command": "close-test",
    "generated": "false",
    "status": Object {
      "enum": "correct",
      "human": "Juist",
    },
  },
  Object {
    "command": "start-test",
    "expected": "false",
  },
  Object {
    "command": "close-test",
    "generated": "false",
    "status": Object {
      "enum": "correct",
      "human": "Juist",
    },
  },
  Object {
    "command": "start-test",
    "expected": "false",
  },
  Object {
    "command": "close-test",
    "generated": "false",
    "status": Object {
      "enum": "correct",
      "human": "Juist",
    },
  },
  Object {
    "command": "start-test",
    "expected": "false",
  },
  Object {
    "command": "close-test",
    "generated": "false",
    "status": Object {
      "enum": "correct",
      "human": "Juist",
    },
  },
  Object {
    "command": "start-test",
    "expected": "false",
  },
  Object {
    "command": "close-test",
    "generated": "false",
    "status": Object {
      "enum": "correct",
      "human": "Juist",
    },
  },
  Object {
    "command": "start-test",
    "expected": "false",
  },
  Object {
    "command": "close-test",
    "generated": "false",
    "status": Object {
      "enum": "correct",
      "human": "Juist",
    },
  },
  Object {
    "command": "start-test",
    "expected": "false",
  },
  Object {
    "command": "close-test",
    "generated": "false",
    "status": Object {
      "enum": "correct",
      "human": "Juist",
    },
  },
  Object {
    "command": "start-test",
    "expected": "false",
  },
  Object {
    "command": "close-test",
    "generated": "false",
    "status": Object {
      "enum": "correct",
      "human": "Juist",
    },
  },
  Object {
    "command": "start-test",
    "expected": "false",
  },
  Object {
    "command": "close-test",
    "generated": "false",
    "status": Object {
      "enum": "correct",
      "human": "Juist",
    },
  },
  Object {
    "command": "start-test",
    "expected": "false",
  },
  Object {
    "command": "close-test",
    "generated": "false",
    "status": Object {
      "enum": "correct",
      "human": "Juist",
    },
  },
  Object {
    "command": "start-test",
    "expected": "false",
  },
  Object {
    "command": "close-test",
    "generated": "false",
    "status": Object {
      "enum": "correct",
      "human": "Juist",
    },
  },
  Object {
    "command": "start-test",
    "expected": "false",
  },
  Object {
    "command": "close-test",
    "generated": "false",
    "status": Object {
      "enum": "correct",
      "human": "Juist",
    },
  },
  Object {
    "command": "start-test",
    "expected": "false",
  },
  Object {
    "command": "close-test",
    "generated": "false",
    "status": Object {
      "enum": "correct",
      "human": "Juist",
    },
  },
  Object {
    "command": "start-test",
    "expected": "false",
  },
  Object {
    "command": "close-test",
    "generated": "false",
    "status": Object {
      "enum": "correct",
      "human": "Juist",
    },
  },
  Object {
    "command": "start-test",
    "expected": "false",
  },
  Object {
    "command": "close-test",
    "generated": "false",
    "status": Object {
      "enum": "correct",
      "human": "Juist",
    },
  },
  Object {
    "command": "start-test",
    "expected": "false",
  },
  Object {
    "command": "close-test",
    "generated": "false",
    "status": Object {
      "enum": "correct",
      "human": "Juist",
    },
  },
  Object {
    "command": "close-testcase",
  },
  Object {
    "command": "start-testcase",
    "description": "Niet geprutst met andere sprites",
  },
  Object {
    "command": "start-test",
    "expected": "false",
  },
  Object {
    "command": "close-test",
    "generated": "false",
    "status": Object {
      "enum": "correct",
      "human": "Correct",
    },
  },
  Object {
    "command": "close-testcase",
  },
  Object {
    "command": "close-context",
  },
  Object {
    "command": "start-context",
  },
  Object {
    "command": "start-testcase",
    "description": "Het ruimteschip staat klaar",
  },
  Object {
    "command": "close-testcase",
  },
  Object {
    "command": "start-testcase",
    "description": "Druk op toets: 1",
  },
  Object {
    "command": "close-testcase",
  },
  Object {
    "command": "close-context",
  },
  Object {
    "command": "start-context",
  },
  Object {
    "command": "start-testcase",
    "description": "Vakje8 bevat kogels",
  },
  Object {
    "command": "start-test",
    "expected": "kogels",
  },
  Object {
    "command": "close-test",
    "generated": "kogels",
    "status": Object {
      "enum": "correct",
      "human": "Correct",
    },
  },
  Object {
    "command": "close-testcase",
  },
  Object {
    "command": "start-testcase",
    "description": "Vakje9 bevat kogels",
  },
  Object {
    "command": "start-test",
    "expected": "kogels",
  },
  Object {
    "command": "close-test",
    "generated": "kogels",
    "status": Object {
      "enum": "correct",
      "human": "Correct",
    },
  },
  Object {
    "command": "close-testcase",
  },
  Object {
    "command": "start-testcase",
    "description": "Vakje10 bevat kogels",
  },
  Object {
    "command": "start-test",
    "expected": "kogels",
  },
  Object {
    "command": "close-test",
    "generated": "kogels",
    "status": Object {
      "enum": "correct",
      "human": "Correct",
    },
  },
  Object {
    "command": "close-testcase",
  },
  Object {
    "command": "start-testcase",
    "description": "Vakje11 bevat kogels",
  },
  Object {
    "command": "start-test",
    "expected": "kogels",
  },
  Object {
    "command": "close-test",
    "generated": "kogels",
    "status": Object {
      "enum": "correct",
      "human": "Correct",
    },
  },
  Object {
    "command": "close-testcase",
  },
  Object {
    "command": "start-testcase",
    "description": "Vakje20 bevat brandstof",
  },
  Object {
    "command": "start-test",
    "expected": "brandstof",
  },
  Object {
    "command": "close-test",
    "generated": "brandstof",
    "status": Object {
      "enum": "correct",
      "human": "Correct",
    },
  },
  Object {
    "command": "close-testcase",
  },
  Object {
    "command": "start-testcase",
    "description": "Vakje21 bevat brandstof",
  },
  Object {
    "command": "start-test",
    "expected": "brandstof",
  },
  Object {
    "command": "close-test",
    "generated": "brandstof",
    "status": Object {
      "enum": "correct",
      "human": "Correct",
    },
  },
  Object {
    "command": "close-testcase",
  },
  Object {
    "command": "start-testcase",
    "description": "Vakje22 bevat brandstof",
  },
  Object {
    "command": "start-test",
    "expected": "brandstof",
  },
  Object {
    "command": "close-test",
    "generated": "brandstof",
    "status": Object {
      "enum": "correct",
      "human": "Correct",
    },
  },
  Object {
    "command": "close-testcase",
  },
  Object {
    "command": "start-testcase",
    "description": "Vakje23 bevat brandstof",
  },
  Object {
    "command": "start-test",
    "expected": "brandstof",
  },
  Object {
    "command": "close-test",
    "generated": "brandstof",
    "status": Object {
      "enum": "correct",
      "human": "Correct",
    },
  },
  Object {
    "command": "close-testcase",
  },
  Object {
    "command": "close-context",
  },
  Object {
    "command": "close-context",
  },
  Object {
    "command": "close-tab",
  },
  Object {
    "command": "close-judgement",
  },
]
`;
